lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -259,6 +259,131 @@ class ExportCommand < HammerCLIKatello::SingleResourceCommand
       end
     end
 
+    class ExportCommand < HammerCLIKatello::SingleResourceCommand
+      include HammerCLIForemanTasks::Async
+
+      action :export
```

Comment:
> We do not use that export API at all for new export/import. On the to do list to deprecate it.

This is why I'm confused, I guess.

Reply:
I'll be honest, I just assumed this was boilerplate. If not using that API, does this line of code go away?

Reply:
You definitely don't need it if you're not using hammer's API resource mechanism, though you may need to override other methods (execute may be sufficient, which you're already overriding), though the ID resolution code uses this as well. Ultimately, you should remove it and see what breaks - it should be code that hammer is running that you're not using for this snowflake command anyway and so should arguably be stubbed.